cms/templates/settings.html

```diff
@@ -306,6 +306,21 @@ <h2 class="title-2">${_("Requirements")}</h2>
                   <input type="text" class="short time" id="course-effort" placeholder="HH:MM" />
                   <span class="tip tip-inline">${_("Time spent on all course work")}</span>
                 </li>
+                % if settings.FEATURES.get('ENABLE_PREREQUISITE_COURSES') and settings.FEATURES.get('MILESTONES_APP'):
```

Comment:
@cahrens the MVP deliverable is to Open edX which uses the Open edX about page (not Drupal). I believe follow-on work will be done to make the feature work with edx.org (Drupal supported), but that is TBD. @marcotuts am I right?